README.md

```diff
@@ -0,0 +1,82 @@
+# bunyan-env-config
+
+A small configuration wrapper around Bunyan logging.  This module's main
+purpose is to contain all the taskcluster boiler plate and configuration logic
+around logging with Bunyan.
+
+We want to use Bunyan logging in our applications so that our logs are easy to
+parse and obtain insights from.  When we use bunyan, we can start putting lots
+of things in our logs easily and use those to monitor our systems.  We can also
+submit things to security monitoring systems a lot easier if we can combine the
+logging messages with the security forwarding.
+
+One of the complexities of using bunyan is that it's a very configurable library.
+Our usage is currently limited to printing to standard output and forwarding
+that output to a log aggregator.
+
+## Log Level Configuration
+write this when we finalise the format
+
+## Api
+The logging object returned by this module is just a bunyan object with a
+debugCompat property on the root logger.  All bunyan APIs after the initial
+configuration continue to apply.  When reading bunyan docs, this library
+replaced only the bunyan.createLogger call.
+
+## Usage
+The intention is for each application to only call the function exported by
+this module once.  In a single js-file library, just do that in your code
+directly.  In larger applications, the suggested method is to create a small
+file (`lib/log.js`) file which sets up the logging for your whole application.
```

Comment:
FYI, I really want to load the logger in `taskcluster-lib-loader` that seems pretty ideal to me :)